keras_core/layers/merging/add.py

```diff
@@ -32,7 +33,7 @@ class Add(Merge):
     def _merge_function(self, inputs):
         output = inputs[0]
         for i in range(1, len(inputs)):
-            output = output + inputs[i]
+            output = ops.add(output, inputs[i])
```

Comment:
Very strange that TF cannot `+` sparse tensors...

Reply:
Indeed

```
      3 x = tf.SparseTensor(indices=[[0, 0], [1, 2]], values=[2, 3], dense_shape=(3, 3))
----> 4 x + x

TypeError: unsupported operand type(s) for +: 'SparseTensor' and 'SparseTensor'
```

Only `__div__`, `__mul__`, `__truediv__` are supported: https://www.tensorflow.org/api_docs/python/tf/sparse/SparseTensor